File: C++/Persona.cpp
#include "Persona.h"

Persona::Persona(const std::string& cedula, const std::string& nombre, const std::string& correo)
    : cedula(cedula), nombre(nombre), correo(correo) {}

std::string Persona::getCedula() const {
    return cedula;
}

std::string Persona::getNombre() const {
    return nombre;
}

std::string Persona::getCorreo() const {
    return correo;
}

File: Java/sga/model/Persona.java
package sga.model;

public abstract class Persona {
    protected String cedula;
    protected String nombre;
    protected String correo;

    public Persona(String cedula, String nombre, String correo) {
        this.cedula = cedula;
        this.nombre = nombre;
        this.correo = correo;
    }

    public String getCedula() {
        return cedula;
    }

    public String getNombre() {
        return nombre;
    }

    public String getCorreo() {
        return correo;
    }

    // Método abstracto - será implementado por las subclases
    public abstract String mostrarInformacion();
}